docs/v2/guides/extending-timber.md

```diff
@@ -573,7 +573,7 @@ function whateverify( $text ) {
 This can now be called in your twig files with:
 
 ```twig
-<h2 id="{{ post.title|slugify }}">{{ post.title|whateverify }}</h2>
```

Comment:
Are we using spaces for filters officially or do we follow the Twig syntax that is used in the Twig documentation?

Reply:
My apologies. I thought spaces were Twig's syntax, but I looked this morning and you're correct — no spaces. I was up too late!

Reply:
I feel you, especially in these times 😴.